generators/server/templates/src/main/java/package/client/OAuth2InterceptedFeignConfiguration.java.ejs

```diff
@@ -46,13 +46,11 @@ public class OAuth2InterceptedFeignConfiguration {
 import java.io.IOException;
 
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
 
 import feign.RequestInterceptor;
 
 import <%=packageName%>.security.oauth2.AuthorizationHeaderUtil;
 
-@Configuration
```

Comment:
Nice catch, I missed this in https://github.com/jhipster/generator-jhipster/issues/7756